templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %}
+{% block content %}
+  <h1>Delete car:</h1>
+  <h1>Are you sure about deleting car info?</h1>
```

Comment:
What if you chose the wrong car to delete? Maybe it's better to add the car brand?